packages/text-annotator/src/model/core/TextAnnotation.ts

```diff
@@ -26,4 +28,5 @@ export interface TextSelector {
 
   offsetReference?: HTMLElement;
 
+  scope?: string;
```

Comment:
Those props shouldn't be present in the core models. They are intentionally used only on the crosswalk with the `in` checks. See the following comments from RSimon:
- https://github.com/recogito/text-annotator-js/pull/170#issuecomment-2452255369
- https://github.com/recogito/text-annotator-js/pull/170#issuecomment-2452277443
- https://github.com/recogito/text-annotator-js/pull/170#issuecomment-2452296112

So I'd prefer keeping the code consistent with the remote to reduce the number of conflicts.